distribution/src/main/packaging/scripts/postinst

```diff
@@ -61,9 +61,9 @@ if [ "x$IS_UPGRADE" != "xtrue" ]; then
 
     elif command -v chkconfig >/dev/null; then
```

Comment:
I'm not entirely sure why, but running `sudo rpm -i elasticsearch...rpm` finds `chkconfig`, because the output from this command is the block below.  However, running `sudo chkconfig --add elasticsearch` fails because `chkconfig` is not found.

I'm not sure what you mean by using `command -p` but running `command -p chkconfig` as a user doesn't find `chkconfig` and running `sudo command -p chkconfig`  doesn't find `command`!